src/messages/Image.cpp

```diff
@@ -459,6 +493,13 @@ void Image::actuallyLoad()
         .execute();
 }
 
+void Image::expireFrames()
+{
+    assertInGuiThread();
+    this->frames_->clear();
```

Comment:
Why add `Frames::clear` when we can reset the unique_ptr instead?

Reply:
I didn't want to call `reset` because `this->frames_` is always expected to be non-null in the case of non-empty images. I also didn't want to do `this->frames_ = std::make_unique<detail::Frames>()` because it does an unnecessary allocation that will just be overwritten when we load the image again.